### DISCUSSION TITLE
cannot use dictionary de-alt

### DISCUSSION THREAD

**Discussion by pulsatilla52** [minad/jinx#178] [Q&A] [answered]
First of all, thank you for your good work.
For writing my texts and my book in emacs/latex I use the old german spelling de-alt with aspell, which is set as default in .emacs. But jinx always uses de_DE as the new spelling methode. How can I make jinx use de-alt? Configuring nuspell with the old dic did work on the cmd-line. But jinx configs itself always  with aspell de_DE.  Any help appreciated.
Erich

**Comment by minad** [comment]
You can change the language with the command `jinx-languages` (select among available dictionaries) or set the variable `jinx-languages` manually.

**Reply by pulsatilla52** [reply]
thank you for your answer!
If I set aspell to 'de-alt' aspell works.

I tested jinx again with a minimal dot-emacs.
Butjinx it doesn't work with de-alt - (and there is a great difference between the new and the old german spelling.)
Setting jinx-language to 'de-alt' changes nothing and 
even if I set:
%%% Local Variables:
%%% TeX-master: "wlnm-master.tex"
%%% jinx-languages: "de-alt"
%%% End:
jinx always uses: Suggestions from dictionary 'de',
the new spelling.
Help as always very appreciated.

**Reply by pulsatilla52** [reply, marked as answer]
I found the solution: jinx cannot work with a dictionary named de-alt.
When I softlinked all 'de-alt' to de_XX.dat,  de_XX.multi etc in the dir where aspell saved its dics, jinx began to work as it should.
Personal dic is saved in ~/.config/enchant/   as   de_XX.dic. (Path in Arch-Linux). 
Best Erich

**Reply by minad** [reply]
Maybe you observed issue https://github.com/AbiWord/enchant/issues/318?
